model/Metadata/Service/ClassMetadataSearcher.php

```diff
@@ -64,15 +64,10 @@ public function findAll(ClassMetadataSearchInput $input): ClassCollection
 
     private function getSubProperties(string $classUri, array $properties = []): void
     {
-        $result = $this->executeQuery('parentClass', $classUri);
+        foreach ($this->getClass($classUri)->getSubClasses(true) as $subclass) {
```

Comment:
Are you sure you are indexing all the classes subclasses without repetition and covering all the cases?

Reply:
I'm listing all classes below my current class. They are all unique. For each of those classes I'm querying elastic search to check property list.